src/pages/authTeachers/Show.tsx

```diff
@@ -32,10 +32,15 @@ const AuthorizedTeacherShow = () => {
     const [subjectData, setSubjectData] = useState<Subject[]>([]);
 
     const fetchData = () => {
-        dataProvider.getOne<Teacher>(MAPPING.TEACHERS, { id: authorizedTeacher?.id }).then((e) => {
-            setClassroomData(Object.values(e.data.classrooms));
-            setSubjectData(Object.values(e.data.classrooms).map((f) => f.subject));
-        });
+        dataProvider
+            .getOne<Teacher>(MAPPING.TEACHERS, { id: authorizedTeacher?.id })
+            .then((e) => {
+                setClassroomData(Object.values(e.data.classrooms));
+                setSubjectData(Object.values(e.data.classrooms).map((f) => f.subject));
+            })
+            .catch(() => {
```

Comment:
We should not display console.log in prod. Is this an error that breaks prod?

Reply:
hey, please look into the new commit